app/graphql/filters/available_pairings.rb

```diff
@@ -1,6 +1,6 @@
 require 'search_object'
 require 'search_object/plugin/graphql'
-  class Resolvers::AvailablePairings
+  class Filters::AvailablePairings
```

Comment:
What does this do?

Reply:
I will take them out. Since the search_object gem is installed in the app they actually don't seem to be necessary.